### PR TITLE
Add automation requirements

### PR DESCRIPTION
**Pull request by Jibec**
To be able to make use of package automated tests, a few files were missing.
I took the opportunity to move additional imports of bash scripts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## Overview
+Zusam (/tsuˈzam/) is a free and open-source way to self-host private forums for groups of friends. Composed of a server written in PHP exposing a REST API and a lightweight webapp, Zusam is extensible and easy to install.
+The goal is to make a stable, extensible, lightweight and user-friendly way to self-host private social groups.
+
+**Shipped version:** 0.3.1
+
+## Links
+
+ * Report a bug about this package: https://github.com//zusam_ynh
```
Comment (nrobinaubertin) [inline README.md]: one "/" too much

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ * Report a bug about Weblate itself: https://github.com/nrobinaubertin/zusam_ynh
```
Comment (nrobinaubertin) [inline README.md]: it's not weblate

File: README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ * Documentation: https://github.com/zusam/zusam/tree/master/documentation
+ * Weblate website: https://zusam.org
```
Comment (nrobinaubertin) [inline README.md]: s/Weblate/Zusam

File: check_process [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+	Level 4=0
+# Level 5:
+	Level 5=auto
+	Level 6=auto
+	Level 7=auto
+	Level 8=0
+	Level 9=0
+	Level 10=0
+;;; Options
+Email=zusam@niels.fr
```
Comment (nrobinaubertin) [inline check_process]: git@niels.fr

File: README.md (hunk 1 above)
Comment (nrobinaubertin) [reply README.md]: Actually, it's https://github.com/nrobinaubertin/zusam_ynh

**Comment by Jibec** [comment]
thanks for the review, every requested changes are done

**Pull request merged**